runtime/lib/component/custom-config.js

```diff
@@ -3,13 +3,21 @@ var logger = require('logger')('custom-config-runtime')
 var config = require('/etc/yoda/custom-config.json')
 var querystring = require('querystring')
 var safeParse = require('@yoda/util').json.safeParse
+var EventEmitter = require('events')
+var property = require('@yoda/property')
 
 /**
  * handle all custom-config
  */
-class CustomConfig {
+class CustomConfig extends EventEmitter {
   constructor (runtime) {
+    super()
```

Comment:
Do you have test this change on device, super seems need a `this` argument.

Reply:
The constructor of EventEmitter is parameterless, I have tested it is no problem.